disputer/index.js

```diff
@@ -113,8 +113,8 @@ async function run(logger, address, pollingDelay, priceFeedConfig, disputerConfi
     }
   } catch (error) {
     logger.error({
-      at: "Disputer#index🚨",
-      message: "Disputer error",
+      at: "Disputer#index",
```

Comment:
Change made to be consistent with all the other bots. the emoji is always on the `message` line, not the `at` line.